contracts/BosonRouter.sol

```diff
@@ -434,21 +567,21 @@ contract BosonRouter is
         ICashier(cashierAddress).addEscrowAmount{value: msg.value}(msg.sender);
     }
 
+   
     /**
-     * @notice check if _tokenIdSupply mapped to gate contract, 
-     * if it does, deactivate (user,_tokenIdSupply) to prevent double spending
-     * @param _tokenIdSupply    ID of the supply token
+     * @notice Buyer requests/commits to redeem a voucher and receives Voucher Token in return.
+     * Price and deposit is specified in tokens.
+     * @param _tokenIdSupply ID of the supply token
+     * @param _issuer Address of the issuer of the supply token
+     * @param _tokensSent total number of tokens sent. Must be equal to buyer deposit plus price
+     * @param deadline deadline after which permit signature is no longer valide. See EIP-2612
+     * @param vPrice signature component  used to verify the permit on the price token. See EIP-2612
```

Comment:
maybe add "v", "r", "s" as part of definitions
(e.g. `@param vPrice v signature component  used to verify the permit on the price token. See EIP-2612`)
I mean, it is obvious what is should be, just parameter names here are not standard.

Reply:
Included in next commit.